autoscheduler/frontend/src/tests/CourseSelectCard.test.tsx

```diff
@@ -65,3 +64,43 @@ test('Remembers state after collapse', async () => {
   // checks that the card remembers how many elements were checked
   expect(checked2).toEqual(checked1);
 });
+
+test('Changes sections in response to changing course', async () => {
+  // arrange
+  const nodeProps = Object.create(Node.prototype, {});
```

Comment:
This line and the one after it just mock some types that don't exist in the Jest environment but are required for Material-UI Popper elements. TypeScript has to ignore line 72 because otherwise, we would have to define every property expected in a `Range` object, but this allows us to define only the ones that we need.